### PR TITLE
#1693 add error message to Equity Token Offering Terms -> ETO is not under crowdfunding regulations

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/eto/registration/pages/EtoTerms.tsx [hunk 1]
```diff
@@ -99,12 +99,11 @@ const EtoRegistrationTermsComponent: React.SFC<IProps> = ({ readonly, savingData
           />
         </div>
 
-        <FormLabel name="currencies">
```
Comment (desfero) [inline app/components/eto/registration/pages/EtoTerms.tsx]: `FormLabel` should be a part of `FormFieldCheckboxGroup` hence the name `*Field*`

File: app/components/shared/forms/form-field/FormCheckbox.tsx [hunk 2]
```diff
@@ -79,29 +80,34 @@ const RadioButtonComponent: React.SFC<IProps & IInternalProps> = ({
   );
 };
 
+/*
+ * Conditional checkbox.
```
Comment (desfero) [inline app/components/shared/forms/form-field/FormCheckbox.tsx]: Do you agree?

File: app/components/shared/forms/form-field/FormCheckbox.tsx [hunk 3]
```diff
@@ -79,29 +80,34 @@ const RadioButtonComponent: React.SFC<IProps & IInternalProps> = ({
...
+ * Use when there is a need to represent true/false value (checked === true).
+ * If array of values is needed use FormFieldCheckboxGroup
+ * @todo Rename to FormFieldConditional (or something better) as FormCheckbox can be misleading with FormFieldCheckbox
```
Comment (desfero) [inline app/components/shared/forms/form-field/FormCheckbox.tsx]: Do you agree?

File: app/lib/yup-ts.ts [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 export const array = <T extends YTS<any>>(shape: T) => new ArrayYTS(shape);
 export const number = () => new NumberYTS();
 export const boolean = () => new BooleanYTS();
-export const onlyTrue = () =>
+export const onlyTrue = (message?: TTranslatedString) =>
   new BooleanYTS().enhance(v =>
     v.test(
       "isTrue",
-      "This field must be set to true",
+      message || ("This field must be set to true" as any),
```
Comment (desfero) [inline app/lib/yup-ts.ts]: I tried to move `This field...` to translations but it's not that easy as changing file to `tsx` breaks a couple of lines with generics so need some refactoring to correctly move this.

File: app/components/eto/registration/pages/EtoTerms.tsx (hunk 1 above)
Comment (Moejoe90) [reply app/components/eto/registration/pages/EtoTerms.tsx]: :+1:

File: app/components/shared/forms/form-field/FormCheckbox.tsx (hunk 3 above)
Comment (Moejoe90) [reply app/components/shared/forms/form-field/FormCheckbox.tsx]: :+1:

File: app/lib/yup-ts.ts (hunk 4 above)
Comment (Moejoe90) [reply app/lib/yup-ts.ts]: :+1:

**Pull request merged**